src/utils/hooks/useYouTubeDataAPI.js

```diff
@@ -0,0 +1,87 @@
+import { useState, useEffect, useMemo } from 'react';
+
+const API_KEY = 'AIzaSyAcgmsZB2M27kvgZju-qtHhKiziG5dzYf8'; // benoit.borrel@gmail.com
+// const API_KEY = 'AIzaSyBfCTxfs9EgNkuHKRjful5zvSVVNymV34g'; // benoit@borrel.com
+const API_URL = 'https://www.googleapis.com/youtube/v3/';
+const API_PARAM_PART = 'snippet';
+const API_PARAM_TYPE = 'video';
+const API_PARAM_MAX_RESULTS = 15;
```

Comment:
Env vars should be placed inside a .env file. This is sensitive data, please remove it from your PR.

Reply:
@dianasaurio it has been taken care of in https://github.com/wizelineacademy/react-bootcamp-2020/pull/11/commits/20c4074b8bb9959a37c480adf209ddaf0042147b